tensorflow_cloud/python/core/deploy.py

```diff
@@ -162,6 +162,12 @@ def _create_request_dict(
         )
 
         if machine_config.is_tpu_config(worker_config):
+            # TPU is not support for TF version > 2.1.0
```

Comment:
The error and test should be moved to validate.py file.

Reply:
done